### DISCUSSION TITLE
Change or set order of coverage panels in plotBrowserTrack()

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#577] [Feature Requests]
**Describe the problem that your feature request would address.**
I would like to manually set the order of browser coverage tracks in plotBrowserTrack(). I can specificy which group configuration with the groupBy parameter, but I would like to change the default order of these labels in BrowserTrack. 

**Describe the solution you'd like**
Can you suggest a method to change the order of the "identities" or cluster levels?

**Comment by jgranja24** [comment]
I will try to think of the best way to do this.

**Comment by liviuspenter** [comment]
You can get this to work with an ugly hack: rename the clusters so that their names are preceeded by numbers that indicate your desired order. For example 3_C1, 2_C2, 1_C3 for ordering them as C3, C2, C1.

**Reply by 1221li** [reply]
Hello, I would like to ask how to change the names of C1,C2,C3, etc. to 1_C3,2_C1,3_C2 if this method is OK. Which step should I proceed with?
